config/initializers/devise.rb

```diff
@@ -158,24 +158,24 @@
   # Defines which strategy will be used to lock an account.
   # :failed_attempts = Locks an account after a number of failed attempts to sign in.
   # :none            = No lock strategy. You should handle locking by yourself.
-  config.lock_strategy = :failed_attempts
+  # config.lock_strategy = :failed_attempts
```

Comment:
I think the default is `:failed_attempts`, but it doesn't matter if the lockable module is not enabled.